bin/secrets/restore.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set +x
+
+if [ -z "$1" ]; then
+    echo "Secrets tree path given as first argument, exiting"
+    exit 1
+
+fi
+
+if [ "${1: -1}" = "/" ]; then
+    SECRETS_TREE_PATH="${1::-1}"
+else
+    SECRETS_TREE_PATH="${1}"
+fi
+ENVIRONMENT=${2:-remote}
+
+. ./bin/secrets/common.sh
+
+remove_base_dir(){
+    target_dir="$1"
+    echo ${target_dir#${SECRETS_TREE_PATH}/secret}
```

Comment:
I'm failing to understand how this removes the base dir. It's only an echo command, isn't it?